tests/components/yamaha/test_media_player.py

```diff
@@ -16,6 +16,7 @@
 def _create_zone_mock(name, url):
     zone = MagicMock()
     zone.ctrl_url = url
+    zone.surround_programs = []
```

Comment:
This is written to entity registry, if not set here we try to write a `Mock` which is caught during test shutdown